src/virtualenv/create/creator.py

```diff
@@ -90,6 +92,13 @@ def add_parser_arguments(cls, parser, interpreter, meta, app_data):
             help="remove the destination directory if exist before starting (will overwrite files otherwise)",
             default=False,
         )
+        parser.add_argument(
+            "--no-vcs-ignore",
```

Comment:
``VIRTUALENV_NO_VCS_IGNORE`` and ``no_vcs_ignore`` will be the names as documented under https://virtualenv.pypa.io/en/latest/cli_interface.html#environment-variables. The destination variable doesn't matter only the CLI flag.